test.rb

```diff
@@ -0,0 +1 @@
+pkg.environment "THING", "value"
```

Comment:
Oh, derp, I was using it to test my Ruby rewriter script. I wasn't crazy enough to redo all those environment hashes by hand. :P I'll delete it.